ansible/k8s-vae-shared-storage.yml

```diff
@@ -0,0 +1,43 @@
+# Shared dynamic storage for the VAE
+
+# The gluster client is required for the two stage heketi storage setup
+- hosts: "{{ idr_environment | default('idr') }}-k8svae-hosts"
```

Comment:
The naming here confuses me. In general, we probably need to talk about how the parallel versioning occurs, so that we need to track both the cluster version and the vae version.

Reply:
The VAE version will be tracked in https://github.com/IDR/k8s-analysis-deploy and not this repo